flexible-classroom/reference/classroom-api.mdx

```diff
@@ -0,0 +1,1880 @@
+---
+title: 'Classroom API'
```

Comment:
Suggestion: Rename this title to "Classroom Cloud Service RESTful API".
Reason: As I explained in the comments on " Classroom SDK", we can use a clear title to make it easier for users to understand what's the differences between "Classroom SDK" and "Classroom API" docs. This is important.

Reply:
As before. Staying as is.

Reply:
Changed in Classroom REST API.

A rest API in its definition goes to something in the cloud, so it is a tautology.